### ISSUE TITLE
When searching, show hits from older releases

### ISSUE THREAD

**Issue by holly-cummins** [quarkusio/code.quarkus.io#1196]
**Description**
(Describe the feature here.)

I really like how if you search but your filters are too restrictive, the site gives you a hint about where to find the thing you're looking for. 

<img width="810" alt="image" src="https://user-images.githubusercontent.com/11509290/235125025-892fd807-bbc5-4afb-92c6-057cdcba4bb9.png">

It would be nice to do this across versions, too, since sometimes extensions drop out of the latest version of the platform. For example, if I really wanted to use kogito, this doesn't help me: 

<img width="943" alt="image" src="https://user-images.githubusercontent.com/11509290/235125213-adca7e60-2f93-46cc-82e1-028c7c07c514.png">

I'd have to somehow know that kogito was in older releases and had not made it into the latest release and manually adjust my search. It could also work in the opposite direction, so that if I specified an older release, but an extension that's only available in the latest release, it would give me a hint how to change my search.

**Comment by maxandersen** [comment]
That's what extensions site should do. Code.quarkus targets specific platform release.

**Comment by holly-cummins** [comment]
I'm not sure targeting a specific platform release rules out helping users if they're trying to target the wrong release for what they want to do. 

I agree that code.quarkus shouldn't mix results across all releases, as the extensions site does, but I think giving a hint to users who've got stuck trying to generate an app would have more positive effects than negative ones.

**Comment by maxandersen** [comment]
Yes. Not disagreeing here but could maybe be solved at least intermediately with a link to extensions.quarkus.io search?
